lib/mcb/commands/geocode.rb

```diff
@@ -0,0 +1,40 @@
+name "geocode"
```

Comment:
@misaka - I accept potentially this is going against the grain slightly, i.e. putting the entire geocode command within the commands` directory. This was to prevent code duplication i.e.
`bin/mcb geocode -c "Site"` or `bin/mcb geocode -c "Provider"`

rather than:

`bin/mcb sites geocode` or `bin/mcb providers geocode`

... but let me know if you prefer another way of doing this.

Reply:
Ya, I think that if we really wanted to avoid code duplication, it should be a service. But here, the code that would be duplicated is so small, and then this method is inflated a bit to handle the class abstraction, I really wonder if it's worth it.